src/emm/emm_cli.py

```diff
@@ -213,7 +213,7 @@ def patch(ctx: click.Context) -> None:
     Any options passed to this command was be forwarded to the `evergreen patch` command. However,
     the following options are already included and should not be added:
 
-    * --skip_confirm, --yes, -y
+    * --yes, -y
```

Comment:
Removed skip_confirm since it's no longer an option in evergreen patch.

```
NAME:
   evergreen patch - submit a new patch to evergreen

USAGE:
   evergreen patch [command options] [arguments...]

OPTIONS:
   --project value, -p value            specify the name of an existing evergreen project
   --finalize, -f                       schedule tasks immediately
   --variants value, -v value           variant names
   --param value                        specify a parameter as a KEY=VALUE pair
   --browse                             open patch url in browser
   --sync_variants value                build variants to sync when task ends
   --sync_tasks value                   tasks to sync when task ends
   --sync_statuses value                filter task statuses for which task sync should run when task ends ('success' or 'failed')
   --sync_timeout value                 max timeout on task sync when task ends (e.g. 15m, 2h) (default: 1h0m0s)
   --large, -l                          enable submitting larger patches (>16MB)
   --yes, -y                            skip confirmation text
   --ref REF                            diff with REF (default: "HEAD")
   --uncommitted, -u                    include uncommitted changes
   --tasks value, -t value              task names
   --alias value, -a value              patch alias (set by project admin)
   --description value, -d value        description for the patch
   --verbose                            show patch summary
   --trigger-alias value                patch trigger alias (set by project admin) specifying tasks from other projects
   --preserve-commits                   preserve separate commits when enqueueing to the commit queue
```